docs/calendar-review.md

```diff
@@ -47,9 +47,27 @@ Examples of calendar-sensitive operations include:
 In other words, units of time greater than a solar day are inherently dependent on the calendar system.
 The only universal solution to date arithmetic would be to omit the concept of months and years, but a date API without months and years would be insufficient for many use cases.
 
+### Why Are Calendars a Higher Priority for ECMAScript vs. Other Platforms?
+
+Problems integrating calendar systems into Temporal are obvious, especially the "Unexpected Calendar Problem" ([see below](#the-unexpected-calendar-problem)), where code breaks when presented with a calendar it wasn’t designed to handle.
+What's less clear are the benefits of integrating calendars in Temporal that can outweigh those problems.
+One way to understand these benefits is to highlight differences between the needs of the ECMAScript ecosystem compared to other platforms like Java and .NET which have taken a different approach to handling calendars.
+Some of these differences include:
+
+- As is well described in the [notes of Java's design of its date/time API](https://github.com/ThreeTen/threeten/wiki/Multi-calendar-system), most code in enterprise apps uses the ISO calendar.
```

Comment:
```suggestion
- As is well described in the [notes of Java's design of its date/time API](https://github.com/ThreeTen/threeten/wiki/Multi-calendar-system), most of the current code in enterprise apps uses the ISO calendar.
```

Reply:
I don't think it makes sense to make this change. It's already watered down from the previous wording. Keep in mind that one of the goals of this doc is to show that we're realistic about where non-ISO calendars are likely to be used (consumer apps in emerging markets) vs. not very likely (business apps everywhere). I think keeping the language as-is will better match reality and will avoid eroding our credibility.